leaks detected:Reflect using node v6.0.0

When running a lab test with v6.0.0 with the latest Lab (v10.3.1), I get the following message:
```
$ node --version
v6.0.0
$ npm --version
3.8.6
$ nvm --version
0.31.0

The following leaks were detected:Reflect
(node:2983) DeprecationWarning: 'root' is deprecated, use 'global'
npm ERR! Test failed.  See above for more details.
```
Works with all 4.x.x, 5.x.x versions of Node.

The failing build can be found here:
https://travis-ci.org/cpilsworth/aws-s3-signurl/jobs/126106138
and a working build here:
https://travis-ci.org/cpilsworth/aws-s3-signurl/jobs/126106139

--- a/lib/leaks.js
+++ b/lib/leaks.js
@@ -81,8 +81,6 @@ exports.detect = function (customGlobals) {
         }
     }
 
-    // Harmony features.
-
     if (global.Promise) {
         whitelist.Promise = true;
     }
@@ -109,6 +107,10 @@ exports.detect = function (customGlobals) {
 
     if (global.WeakSet) {
         whitelist.WeakSet = true;
+    }
+
+    if (global.Reflect) {
+        whitelist.Reflect = true;
     }
 
     if (global.DTRACE_HTTP_SERVER_RESPONSE) {

--- a/test/leaks.js
+++ b/test/leaks.js
@@ -10,7 +10,7 @@ const Lab = require('../');
 // Declare internals
 
 const internals = {
-    harmonyGlobals: ['Promise', 'Proxy', 'Symbol', 'Map', 'WeakMap', 'Set', 'WeakSet']
+    harmonyGlobals: ['Promise', 'Proxy', 'Symbol', 'Map', 'WeakMap', 'Set', 'WeakSet', 'Reflect']
 };
 
 

